kopi/assetlist.json

```diff
@@ -0,0 +1,282 @@
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/non-cosmos/ethereum/images/usdt.svg"
+        }
+      ],
+      "keywords": [
+        "dex",
+        "mm"
+      ],
+      "socials": {
+        "twitter": "https://x.com/kopi_money",
+        "discord": "https://discord.gg/kopi-money"
+      },
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "USDC from Noble Chain",
+      "denom_units": [
+        {
+          "denom": "ibc/8E27BA2D5493AF5636760E354E46004562C46AB7EC0CC4C1CA14E9E20E2545B5",
+          "exponent": 0
+        },
+        {
+          "denom": "USDC",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/8E27BA2D5493AF5636760E354E46004562C46AB7EC0CC4C1CA14E9E20E2545B5",
+      "name": "USDC",
+      "display": "USDC",
+      "symbol": "USDC",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kopi/images/usdc.svg"
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/kopi/images/usdc.svg"
+        }
+      ],
+      "keywords": [
+        "dex",
+        "mm"
+      ],
+      "socials": {
+        "twitter": "https://x.com/kopi_money",
+        "discord": "https://discord.gg/kopi-money"
+      },
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "USDT from injective Chain",
+      "denom_units": [
+        {
+          "denom": "ibc/D8A36AE90F20FE4843A8D249B1BCF0CCDDE35C4B605C8DED57BED20C639162D0",
+          "exponent": 0
+        },
+        {
+          "denom": "USDT.inj",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/D8A36AE90F20FE4843A8D249B1BCF0CCDDE35C4B605C8DED57BED20C639162D0",
+      "name": "USDT.inj",
+      "display": "USDT.inj",
+      "symbol": "USDT.inj",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/non-cosmos/ethereum/images/usdt.svg"
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/non-cosmos/ethereum/images/usdt.svg"
+        }
+      ],
+      "keywords": [
+        "dex",
+        "mm"
+      ],
+      "socials": {
+        "twitter": "https://x.com/kopi_money",
+        "discord": "https://discord.gg/kopi-money"
+      },
+      "type_asset": "sdk.coin"
+    },
+    {
+      "description": "injective from injective Chain",
+      "denom_units": [
+        {
+          "denom": "ibc/DE63D8AC34B752FB7D4CAA7594145EDE1C9FC256AC6D4043D0F12310EB8FC255",
+          "exponent": 0
+        },
+        {
+          "denom": "INJ",
+          "exponent": 18
+        }
+      ],
+      "base": "ibc/DE63D8AC34B752FB7D4CAA7594145EDE1C9FC256AC6D4043D0F12310EB8FC255",
+      "name": "INJ",
+      "display": "Injective",
+      "symbol": "Injective",
```

Comment:
I believe these values are backwards. The symbol should be "INJ" and the name should be "Injective"